samples/Basic_NFC/Makefile-user.mk

```diff
@@ -0,0 +1,40 @@
+## Local build configuration
```

Comment:
Please, replace the Makefile-user.mk with that one `https://github.com/SmingHub/Sming/blob/develop/samples/Basic_Blink/Makefile-user.mk`, Don't put your local version of Basic_Blink here :)